src/views/shop/shoppingCar.vue

```diff
@@ -235,4 +235,11 @@ export default {
 	margin-bottom: 150px;
 	padding-bottom: 20px;
 }
+
```

Comment:
Why this modification ? This is not a part of the main issue

Reply:
Thanks for your advice, I just remove this lines, it was affecting the padding and margin of the products container in the shopping car

Reply:
Maybe this can be a new issue 😃